python/system/ksp-cve-2021-3426-python-pydoc.yaml

```diff
@@ -0,0 +1,28 @@
+# KubeArmor is an open source software that enables you to protect your cloud workload at run-time.
+# To learn more about KubeArmor visit:
+# https://www.accuknox.com/kubearmor/
+
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorPolicy
+metadata:
+  name: ksp-cve-2021-3426-python-pydoc
+  namespace: default    #Change with your namespace
+spec:
+  tags: ["CVE","CVE-2021-3426","Python","Pydoc"]
+  message: "unauthorized user tried to start Pydoc server"
```

Comment:
Change the message, few suggestions:
 - [ ] Uppercase when the sentence starts.
 - [ ] Include CVE in message as well.